test/functional/fundrawtransaction.py

```diff
@@ -452,7 +452,7 @@ def run_test(self):
         self.stop_node(2)
         self.stop_node(3)
         self.nodes[1].encryptwallet("test")
-        bitcoind_processes[1].wait(timeout=BITCOIND_PROC_WAIT_TIMEOUT)
+        self.bitcoind_processes[1].wait(60)
```

Comment:
Done. I've also restored the use of a constant here (at least until the next PR in this series is merged).